src/ecdsa/util.py

```diff
@@ -312,6 +313,7 @@ def sigdecode_string(signature, order):
     :return: tuple with decoded 'r' and 's' values of signature
     :rtype: tuple of ints
     """
+    signature = normalise_bytes(signature)
```

Comment:
In keys.py you didn't normalize the signature, but here you are. A comment about why this is a different case than when signature is passed to the verify function would probably be in order.